src/core/sagas/site.js

```diff
@@ -4,20 +4,22 @@ import { call, put, select, takeLatest } from 'redux-saga/effects';
 import { getSiteStatus } from 'core/api/site';
 import { FETCH_SITE_STATUS, loadSiteStatus } from 'core/reducers/site';
 import { getState } from 'core/sagas/utils';
+import log from 'core/logger';
 import type { GetSiteStatusParams } from 'core/api/site';
 import type { Saga } from 'core/types/sagas';
 
-// This saga is not triggered by the UI but on the server side, hence do not
-// have a `errorHandler`. We do not want to miss any error so we let this saga
-// throw errors without catching them.
 export function* fetchSiteStatus(): Saga {
   const state = yield select(getState);
 
   const params: GetSiteStatusParams = { api: state.api };
 
-  const { read_only: readOnly, notice } = yield call(getSiteStatus, params);
+  try {
+    const { read_only: readOnly, notice } = yield call(getSiteStatus, params);
 
-  yield put(loadSiteStatus({ readOnly, notice }));
+    yield put(loadSiteStatus({ readOnly, notice }));
+  } catch (error) {
+    log.error(`Could not fetch site status: ${error}`);
```

Comment:
Should we consider using an errorHandler that would allow something to appear on the site when this happens, or is that not something we do for server-executed sagas?

Reply:
I don't think we have that for server-executed sagas indeed. I am not sure if that's really a problem for site status but it will be something to consider for #9421